controllers/index.js

```diff
@@ -1,45 +1,37 @@
-'use strict';
```

Comment:
I get the following error due to this removal `Block-scoped declarations (let, const, function, class) not yet supported outside strict mode` I think upgrading the version of nodejs will likely fix this, but I would like the app to support the LTS version of nodejs so would like to stick with the node v6 minimum. I think the PR can be merged, but could you just put back the `use strict`?